### DISCUSSION TITLE
Thumbs down

### DISCUSSION THREAD

**Discussion by bob420-svg** [community/community#66128] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I officially don't like github. I'm having too much trouble with this app on trying to learn to use it.

**Comment by seyLu** [comment]
#### Hi @cghost420 

It takes a while to learn something, and be good enough at it to start appreciating it. And that's true for anything really, not just learning Github.

I suggest to take a crash course: [How to use Git and Github](https://www.youtube.com/watch?v=RGOj5yH7evk), then [Git for Professionals](https://www.youtube.com/watch?v=Uszj_k0DGsg). Optionally, you can also take [Advanced Git](https://www.youtube.com/watch?v=qsTthZi23VE).

And if you want to explore a Github feature and deep dive into it, the official docs is a good place to start. If you want to learn how to collaborate in Github with other developers, you can start with this vid [A typical Github Workflow](https://www.youtube.com/watch?v=02aQhH5cNBg) or this vid [Github Flow](https://www.youtube.com/watch?v=juLIxo42A_s) and learn more about it using the official docs for [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow).

**Comment by bob420-svg** [comment]
Thank you for that. I told git to take the comments out but I guess I
didn't do it right. I'll take then out manually

**Comment by dennybill** [comment]
> #### Hi @cghost420
> It takes a while to learn something, and be good enough at it to start appreciating it. And that's true for anything really, not just learning Github.
> 
> I suggest to take a crash course: [How to use Git and Github](https://www.youtube.com/watch?v=RGOj5yH7evk), then [Git for Professionals](https://www.youtube.com/watch?v=Uszj_k0DGsg). Optionally, you can also take [Advanced Git](https://www.youtube.com/watch?v=qsTthZi23VE).
> 
> And if you want to explore a Github feature and deep dive into [it](https://apkinu.com/), the official docs is a good place to start. If you want to learn how to collaborate in Github with other developers, you can start with this vid [A typical Github Workflow](https://www.youtube.com/watch?v=02aQhH5cNBg) or this vid [Github Flow](https://www.youtube.com/watch?v=juLIxo42A_s) and learn more about it using the official docs for [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow).
GitHub's official documentation is an excellent resource to explore specific GitHub features and functionalities in depth.
It provides detailed information on using GitHub for version control, project management, and collaboration.
